docs/console-application-issues.md

```diff
@@ -17,9 +17,9 @@ ms.assetid: a561fbdd-b50d-4687-92d7-735377a7991d
 
 # Console Application Issues
 
-The 8-bit console functions use the OEM code page. All other functions use the ANSI code page by default. This means that strings returned by the console functions may not be processed correctly by the other functions and vice versa. For example, if **FindFirstFileA** returns a string that contains certain extended ANSI characters, **WriteConsoleA** will not display the string properly.
+The 8-bit console functions use the OEM code page. All other functions use the ANSI code page by default. This means that strings returned by the console functions may not be processed correctly by the other functions and vice versa. For example, if `FindFirstFileA` returns a string that contains certain extended ANSI characters, `WriteConsoleA` will not display the string properly.
 
-The best long-term solution for a console application is to use **[Unicode](https://docs.microsoft.com/windows/win32/intl/unicode)**. The console will accept UTF-16 encoding on the W variant of the APIs or UTF-8 encoding on the A variant of the APIs after using **[SetConsoleCP](setconsolecp.md)** and **[SetConsoleOutputCP](setconsoleoutputcp.md)** to `65001` (`CP_UTF8` constant) for the UTF-8 code page.
+The best long-term solution for a console application is to use `[Unicode](https://docs.microsoft.com/windows/win32/intl/unicode)`. The console will accept UTF-16 encoding on the W variant of the APIs or UTF-8 encoding on the A variant of the APIs after using `[SetConsoleCP](setconsolecp.md)` and `[SetConsoleOutputCP](setconsoleoutputcp.md)` to `65001` (`CP_UTF8` constant) for the UTF-8 code page.
 
 Barring that solution, a console application should use the [SetFileApisToOEM](https://msdn.microsoft.com/library/windows/desktop/aa365534) function. That function changes relevant file functions so that they produce OEM character set strings rather than ANSI character set strings.
```

Comment:
This is an untranslatable name too... `SetFileApisToOEM`. I wonder if there are some in here that are not bolded already.